src/main/java/org/broadinstitute/consent/http/db/mapper/DatasetReducer.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.consent.http.db.mapper;
+
+import org.broadinstitute.consent.http.models.DataSet;
+import org.broadinstitute.consent.http.models.DataUse;
+
+import org.jdbi.v3.core.result.LinkedHashMapRowReducer;
+import org.jdbi.v3.core.result.RowView;
+import java.util.Objects;
+
+import java.util.Map;
+
+public class DatasetReducer implements LinkedHashMapRowReducer<Integer, DataSet> {
+  @Override
+  public void accumulate(Map<Integer, DataSet> map, RowView rowView) {
+    DataSet dataset = map.computeIfAbsent(
+      rowView.getColumn("datasetid", Integer.class),
+      id -> rowView.getRow(DataSet.class));
+    
+      if(Objects.nonNull(rowView.getColumn("dataUse", String.class))) {
```

Comment:
nit: spacing, here and throughout. Your IDE should let you configure this. We use Google's java style guide referenced here:
* https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md
* https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Here it is in Intelli-j
<img width="1012" alt="Screen Shot 2021-09-14 at 8 39 17 AM" src="https://user-images.githubusercontent.com/116679/133259041-50c517c3-12ca-4b6a-91d5-63a2fa7765ea.png">